deps/build.jl

```diff
@@ -0,0 +1,27 @@
+rootdir = dirname(@__DIR__)
```

Comment:
This PR should absolutely not be merged with this. I have no idea what it is trying to do but it is completely wrong and will mess up the projects for users (even in environments they don't have active right now!).

I have said over and over that doing these types of Pkg-shenanigans in the build script is incorrect and will not work properly. However, since I am clearly getting ignored, I'll just stop.

Reply:
Dear @KristofferC, I do not like or want to do this, but I have any other solution. All of my tests simply fail. I am experimenting with brute-force solutions to see where the problem comes from.